scipy/stats/tests/test_continuous_basic.py

```diff
@@ -128,6 +128,9 @@ def cases_test_cont_basic():
     for distname, arg in distcont[:] + [(histogram_test_instance, tuple())]:
         if distname == 'levy_stable':
             continue
+        elif distname == "gilbrat":
+            # functionality tested through gibrat; warning tested separately
+            continue
```

Comment:
I'd suggest to remove the deprecated `gilbrat` alias from the list, https://github.com/scipy/scipy/blob/main/scipy/stats/_distr_params.py#L47 , instead. Less code churn this way